setup.py

```diff
@@ -19,7 +19,6 @@
     "prettytable",
     # IPython dropped support for Python 3.8
     "ipython<=8.12.0; python_version <= '3.8'",
-    "ipython",
```

Comment:
why is ipython removed?

Reply:
@edublancas suggested the removal
Is there any issue you faced in your manual tests?